docs/whatsnew/6.0.rst

```diff
@@ -98,6 +99,27 @@ format::
     -------- ------------ ------- ------- ------- ----------- -------
     Planck18        67.66 0.30966  2.7255   3.046 0.0 .. 0.06 0.04897
 
+.. _whatsnew-6.0-iers-data:
+
+Updates to how IERS data are handled
+====================================
+
+Some parts of astropy, such as coordinate and time transformations, rely on
+tables from the International Earth Rotation and Reference Systems (IERS)
+service (these are the IERS A, IERS B, and leap second tables). IERS A used
+to always be automatically downloaded, whereas the IERS B and leap second
```

Comment:
Is there supposed to be a dash or not? Previous mentions above have a dash. I think we should be consistent one way or another. This comment applies to all other similar occurrences.
```suggestion
service (these are the IERS-A, IERS-B, and leap second tables). IERS-A used
to always be automatically downloaded, whereas the IERS-B and leap second
```